### ISSUE TITLE
Arithmetic Overflow

### ISSUE THREAD

**Issue by Deadpikle** [ControlzEx/ControlzEx#34]
Heyo,

I just got this stack trace while using my software while using the latest MahApps alpha. My software can no longer start. 😅

```
Target Site: ControlzEx.Standard.WS SetWindowStyle(IntPtr, ControlzEx.Standard.WS)
Message: Arithmetic operation resulted in an overflow.
Stack Trace: 
   at ControlzEx.Standard.NativeMethods.SetWindowStyle(IntPtr hWnd, WS dwNewLong) in C:\projects\controlzex\src\ControlzEx\Microsoft.Windows.Shell\Standard\NativeMethods.cs:line 3414
   at MahApps.Metro.Controls.GlowWindow.OnSourceInitialized(EventArgs e) in C:\...\MahApps.Metro\MahApps.Metro.Shared\Controls\GlowWindow.xaml.cs:line 200
   at System.Windows.Window.CreateSourceWindow(Boolean duringShow)
   at System.Windows.Window.CreateSourceWindowDuringShow()
   at System.Windows.Window.SafeCreateWindowDuringShow()
   at System.Windows.Window.ShowHelper(Object booleanBox)
   at System.Windows.Window.Show()
   at MahApps.Metro.Behaviours.GlowWindowBehavior.Show() in C:\...MahApps.Metro\MahApps.Metro.Shared\Behaviours\GlowWindowBehavior.cs:line 253
   at MahApps.Metro.Behaviours.GlowWindowBehavior.AssociatedObjectOnLoaded(Object sender, RoutedEventArgs routedEventArgs) in C:\...\MahApps.Metro\MahApps.Metro.Shared\Behaviours\GlowWindowBehavior.cs:line 132
   at System.Windows.RoutedEventHandlerInfo.InvokeHandler(Object target, RoutedEventArgs routedEventArgs)
   at System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
   at System.Windows.UIElement.RaiseEventImpl(DependencyObject sender, RoutedEventArgs args)
   at System.Windows.UIElement.RaiseEvent(RoutedEventArgs e)
   at System.Windows.BroadcastEventHelper.BroadcastEvent(DependencyObject root, RoutedEvent routedEvent)
   at System.Windows.BroadcastEventHelper.BroadcastLoadedEvent(Object root)
   at MS.Internal.LoadedOrUnloadedOperation.DoWork()
   at System.Windows.Media.MediaContext.FireLoadedPendingCallbacks()
   at System.Windows.Media.MediaContext.FireInvokeOnRenderCallbacks()
   at System.Windows.Media.MediaContext.RenderMessageHandlerCore(Object resizedCompositionTarget)
   at System.Windows.Media.MediaContext.RenderMessageHandler(Object resizedCompositionTarget)
   at System.Windows.Media.MediaContext.Resize(ICompositionTarget resizedCompositionTarget)
   at System.Windows.Interop.HwndTarget.OnResize()
   at System.Windows.Interop.HwndTarget.HandleMessage(WindowMessage msg, IntPtr wparam, IntPtr lparam)
   at System.Windows.Interop.HwndSource.HwndTargetFilterMessage(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndWrapper.WndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndSubclass.DispatcherCallbackOperation(Object o)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
   at System.Windows.Threading.Dispatcher.LegacyInvokeImpl(DispatcherPriority priority, TimeSpan timeout, Delegate method, Object args, Int32 numArgs)
   at MS.Win32.HwndSubclass.SubclassWndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam)
```

**Comment by punker76** [comment]
@Deadpikle Its' fixed with 3.0.1-dev0055 a25990ca4d0194b52fac1e6d150d65fa549542c5

**Comment by Deadpikle** [comment]
@punker76 Ah, thanks! 👍
